src/library/Button/index.d.ts

```diff
@@ -1 +1,36 @@
-declare module '@targetx/mineral-ui/Button';
+import {
+  ElementType,
+  ReactElement,
+  ReactNode,
+  Ref,
+  SyntheticEvent
+} from 'react';
+
+export declare namespace Button {
+  type Size = 'small' | 'medium' | 'large' | 'jumbo';
```

Comment:
if they don't need to be exported then you can move out of namespace and define locally only.